src/wiki/installing-and-updating-drivers.md

```diff
@@ -228,6 +282,7 @@ Safe mode is a diagnostic tool, which loads Windows with only the essential driv
 
 ### Reinstall Windows
 
-If none of the solutions above work, you may need to reinstall Windows. This will erase all your data and applications, so make sure you have a backup. You can learn how to backup your data [here](/wiki/backup.html).
+If none of the solutions above work, you may need to reinstall Windows. This will erase all your data and applications, so make sure you have a backup. You can learn how to back up your data [here](/wiki/backup.html).
```

Comment:
```suggestion
If none of the solutions above work, you may need to reinstall Windows. This will erase all your data and applications, so make sure you have a backup. You can learn how to back-up your data [here](/wiki/backup.html).
```